Classes/Emogrifier.php

```diff
@@ -296,12 +260,59 @@ public function emogrify()
             $this->removeInvisibleNodes($xpath);
         }
 
-        $this->copyCssWithMediaToStyleNode($cssParts, $xmlDocument);
+        $this->copyCssWithMediaToStyleNode($cssParts['media'], $xmlDocument);
 
         return $xmlDocument->saveHTML();
     }
 
     /**
+     * Parse a list of selectors from a string of css.
```

Comment:
Generally, the first sentence in a method documentation should be third person (with "This method" omitted): "Parses a list …"